rebuild/lib/index.js

```diff
@@ -5,6 +5,36 @@
 /* eslint-disable @typescript-eslint/no-unsafe-call */
 const bindings = require("bindings")("blst_ts_addon");
 
+bindings.verify = function verify(msg, pk, sig) {
+  return bindings.aggregateVerify([msg], [pk], sig);
+};
+
+bindings.asyncVerify = function asyncVerify(msg, pk, sig) {
+  return bindings.asyncAggregateVerify([msg], [pk], sig);
+};
+
+bindings.fastAggregateVerify = function fastAggregateVerify(msg, pks, sig) {
+  let key;
+  try {
+    // this throws for invalid key, catch and return false
+    key = bindings.aggregatePublicKeys(pks);
+  } catch {
+    return false;
+  }
+  return bindings.aggregateVerify([msg], [key], sig);
+};
+
+bindings.asyncFastAggregateVerify = function asyncFastAggregateVerify(msg, pks, sig) {
+  let key;
+  try {
+    // this throws for invalid key, catch and return false
+    key = bindings.aggregatePublicKeys(pks);
```

Comment:
is there not an async version for aggregating pub keys?

Reply:
No.  In testing it is much much slower and not worth it.  The overhead to queue the work was >1ms and these are in the 1-2 digit nano seconds in perf testing.  Figured it wasn't worth it but will be happy to make them async if you think they should be.

Reply:
then how much does the aggregateVerify costs? yes its important to see that queuing be a very small fraction so that the final compute perf comes out ahead

Reply:
Here is a table from the perf test in lodestar for it running sync on an older branch:

https://github.com/ChainSafe/lodestar/pull/5633

![Screenshot 2023-08-02 at 10 10 07 PM](https://github.com/ChainSafe/blst-ts/assets/18608739/6a8284b7-7970-46e3-90ae-4e628fe5e58c)

The sync version ran in the micro seconds and the async version ran in the miliseconds is why i switched the perf test to sync.  It was faster.  The delay to que to libuv and wait for the event loop to return the result was greater than a milisecond and on some runs it was 4-5 ms.  The result actually surprised me!!  I was expecting the math to add the points to be relatively slow but it was 3 orders of magnitude faster than waiting for libuv to do the work.